Sources/Danger/Plugins/SwiftLint/SwiftLint.swift

```diff
@@ -30,7 +30,7 @@ public struct SwiftLint {
 
 /// This extension is for internal workings of the plugin. It is marked as internal for unit testing.
 extension SwiftLint {
-    // swiftlint:disable:next function_body_length
+    // swiftlint:disable:next function_body_length function_parameter_count
     static func lint(
```

Comment:
<!--
  0 failure: 
  1 warning:  SwiftLint rule 'f...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/Plugins/SwiftLint/SwiftLint.swift;
  Line: 34;
-->

- :warning: SwiftLint rule 'function_parameter_count' did not trigger a violation in the disabled region. Please remove the disable command. (`superfluous_disable_command`)